doc/classes/ProjectSettings.xml

```diff
@@ -2364,6 +2364,136 @@
+			The amount of collision margin to use for certain convex collision shapes, such as [BoxShape3D], [CylinderShape3D] and [ConvexPolygonShape3D], as a fraction of the shape's shortest axis.
+			[b]Note:[/b] Collision margins in Jolt do not add any extra size to the shape. Instead the shape is first shrunk by the margin and then inflated by the same amount, resulting in a shape with rounded edges. This is mainly used to speed up collision detection with convex shapes.
+			[b]Note:[/b] Setting this too low (e.g. 0) can also negatively affect the accuracy of the collision detection with convex shapes.
+		</member>
+		<member name="physics/jolt_physics_3d/joints/world_node" type="int" setter="" getter="" default="0">
+			Which of the two nodes in a single-body joint (i.e. when omitting one of the nodes) should represent the world. This can be thought of as having the omitted node be a [StaticBody3D] at [code](0, 0, 0)[/code].
+			[b]Note:[/b] Godot Physics uses [code]node_b[/code] as the default.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/max_angular_velocity" type="float" setter="" getter="" default="47.1239">
+			The maximum angular velocity that a [RigidBody3D] can reach.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/max_bodies" type="int" setter="" getter="" default="10240">
+			The maximum number of [PhysicsBody3D] to support at the same time, awake or sleeping.
+			[b]Note:[/b] When this limit is exceeded a warning is emitted and anything past that point is undefined behavior.
+			[b]Note:[/b] This limit also applies within the editor.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/max_body_pairs" type="int" setter="" getter="" default="65536">
+			The maximum number of body pairs to allow processing of.
+			[b]Note:[/b] When this limit is exceeded collisions will randomly be ignored and bodies will pass through each other.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/max_contact_constraints" type="int" setter="" getter="" default="20480">
+			The maximum number of contact constraints to allow processing of.
+			[b]Note:[/b] When this limit is exceeded collisions will randomly be ignored and bodies will pass through each other.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/max_linear_velocity" type="float" setter="" getter="" default="500.0">
+			The maximum linear velocity that a [RigidBody3D] can reach.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/temporary_memory_buffer_size" type="int" setter="" getter="" default="32">
+			The amount of memory to pre-allocate for the stack allocator used within Jolt.
+		</member>
+		<member name="physics/jolt_physics_3d/limits/world_boundary_shape_size" type="float" setter="" getter="" default="2000.0">
+			How wide/deep/high a [WorldBoundaryShape3D] will be.
+			[b]Note:[/b] Only half of this value will be used for its height.
```

Comment:
I rephrased this to instead be:

> [WorldBoundaryShape3D] is not unbounded when using Jolt, in order to prevent precision issues. This setting specifies the size of its bounds, for all three dimensions, with the plane at the center.

I'm not very happy with it. I welcome other suggestions.

I don't believe this defines the AABB, so using the word "bounds" may be misleading here. You can think of it as defining the size of a `BoxShape3D` I suppose, but where the plane sits at the center of the box, with empty space above it.